core/vdbe/mod.rs

```diff
@@ -343,6 +344,8 @@ pub struct Program {
     pub change_cnt_on: bool,
     pub result_columns: Vec<ResultSetColumn>,
     pub table_references: Vec<TableReference>,
+    #[cfg(feature = "json")]
+    pub json_cache: JsonCacheCell,
```

Comment:
I am pretty sure you want to move this to `ProgramState`.

Reply:
fixed